brainzutils/musicbrainz_db/tests/test_release.py

```diff
@@ -39,3 +43,25 @@ def test_fetch_multiple_releases(self):
         self.assertEqual(len(releases), 2)
         self.assertEqual(releases['a64a0467-9d7a-4ffa-90b8-d87d9b41e311']['name'], 'Numb/Encore')
         self.assertEqual(releases['f51598f5-4ef9-4b8a-865d-06a077bf78cf']['name'], 'Collision Course')
+
+    def test_get_releases_using_recording_mbid(self):
+        """Tests if releases are fetched correctly for a given recording MBID"""
+
+        mb_release.recording = MagicMock()
+        mb_release.recording.query.return_value.options.return_value.options.return_value.\
```

Comment:
You should mock `recording.get_recording_by_mbid` here to return the recording you want, and do a `mock_get_recording.assert_called_once_with(mbid)`, makes the test a bit simpler to read.